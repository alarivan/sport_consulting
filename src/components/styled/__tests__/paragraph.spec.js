import React from "react"
import { render } from "@testing-library/react"
import Paragraph from "../Paragraph.css.js"

describe("Paragraph", () => {
  it("renders correctly", () => {
    const { asFragment, container } = render(<Paragraph>Content</Paragraph>)

    expect(container.firstChild).toHaveStyleRule("font-weight", "500")
    expect(container.firstChild).toHaveStyleRule("font-size", "1.2rem")

    expect(asFragment()).toMatchSnapshot()
  })

  it("renders correctly with bold", () => {
    const { asFragment, container } = render(
      <Paragraph bold={true}>Content</Paragraph>
    )

    expect(container.firstChild).toHaveStyleRule("font-weight", "600")
    expect(container.firstChild).toHaveStyleRule("font-size", "1.5rem")

    expect(asFragment()).toMatchSnapshot()
  })
})
