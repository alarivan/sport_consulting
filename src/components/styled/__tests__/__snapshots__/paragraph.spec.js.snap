// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph renders correctly 1`] = `
<DocumentFragment>
  .c0 {
  font-size: 1.2rem;
  font-weight: 500;
  margin-bottom: 1.5rem;
  font-weight: 500;
}

<p
    class="c0"
  >
    Content
  </p>
</DocumentFragment>
`;

exports[`Paragraph renders correctly with bold 1`] = `
<DocumentFragment>
  .c0 {
  font-size: 1.5rem;
  font-weight: 500;
  margin-bottom: 1.5rem;
  font-weight: 600;
}

<p
    class="c0"
  >
    Content
  </p>
</DocumentFragment>
`;
